pkgs/development/python-modules/chart-studio/default.nix

```
@@ -3,53 +3,57 @@
   buildPythonPackage,
   fetchFromGitHub,
   mock,
-  nose,
   plotly,
-  pytest,
   requests,
   retrying,
   setuptools,
-  six,
+  pytestCheckHook,
 }:
 
 buildPythonPackage rec {
   pname = "chart-studio";
-  version = "5.22.0";
+  version = "5.23.0-unstable-2024-07-23";
```

```suggestion
  version = "1.1.0-unstable-2024-07-23";
```

The `version` here is the version of the `plotly` package that happens to share a repository with this; the latest PyPI release is actually [1.1.0](https://pypi.org/project/chart-studio/). We should use the versioning scheme of the actual package to avoid confusing users here.